src/assets/thirdPartyServices/snet/minecraftizing_service/index.js

```diff
@@ -0,0 +1,195 @@
+import { Grid, IconButton, MuiThemeProvider, Tooltip } from "@material-ui/core";
+
+import { MinecraftizingService } from "./MinecraftizingService_pb_service";
+import RandomDropDown from "./DropDown";
+import MethodNamesDropDown from "../../common/MethodNamesDropDown";
+import SNETImageUpload from "../../standardComponents/SNETImageUpload";
+
+// const outsideWrapper = {
+//   width: "256px",
+//   height: "256px",
+//   margin: "0px 0px",
+//   border: "0px",
+// };
+// const insideWrapper = {
+//   width: "100%",
+//   height: "100%",
+//   position: "relative",
+// };
+const coveredImage = {
+  width: "100%",
+  height: "100%",
+  position: "absolute",
+  top: "0px",
+  left: "0px",
+};
+// const coveringCanvas = {
+//   width: "100%",
+//   height: "100%",
+//   position: "absolute",
+//   top: "0px",
+//   left: "0px",
+// };
+
+const initialUserInput = {
+  methodName: "Select a method",
+  model_name: "Select model",
+  dataset: "Select dataset",
+  input_image: undefined,
+};
+
+export default class MinecraftService extends React.Component {
+  constructor(props) {
+    super(props);
+    this.submitAction = this.submitAction.bind(this);
+    this.handleFormUpdate = this.handleFormUpdate.bind(this);
+    this.getImageData = this.getImageData.bind(this);
+
+    this.state = {
+      ...initialUserInput,
+      users_guide: "",
+      code_repo: "",
+      response: undefined,
+    };
+  }
+
+  canBeInvoked() {
+    return this.state.input_image && this.state.network_name !== "Select model" && this.state.dataset !== "Select dataset";
+  }
+
+
+  handleFormUpdate(event) {
+    this.setState({ [event.target.name]: event.target.value });
+  }
+
+  submitAction() {
+    const { methodName, model_name, dataset, input_image } = this.state;
+    const methodDescriptor = MinecraftizingService[methodName];
+    const request = new methodDescriptor.requestType();
+
+    request.setNetworkName(model_name);
+    request.setDataset(dataset);
+    request.setInputImage(input_image);
+
+    const props = {
+      request,
+      onEnd: ({ message }) => {
+        this.setState({
+          ...initialUserInput,
+          response: { status: message.getStatus(), data: message.getOutput(), source: input_image},
+        });
+      },
+    };
+
+    this.props.serviceClient.unary(methodDescriptor, props);
+  }
+
+  getImageData(data)
+  {
+    this.setState({input_image: data})
+  }
+
+  renderForm() {
+    const serviceNameOptions = ["Select a method", ...this.props.serviceClient.getMethodNames(MinecraftizingService)];
+    const modelNames = ["Select model", "UGATIT", "cycle_gan"];
+    const datasetNames = ["Select dataset", "minecraft_landscapes"];
+
+    return (
+      <React.Fragment>
+        <div className="row">
+          <div className="col-md-3 col-lg-3" style={{ padding: "10px", fontSize: "13px", marginLeft: "10px" }}>
+            Method Name:{" "}
+          </div>
+          <div className="col-md-3 col-lg-3">
```

Comment:
we're not using bootstrap anymore. please get rid of the bootstrap classNames. e.g `className="col-md-3 col-lg-3"`